src/bindist/installdos

```diff
@@ -3,6 +3,8 @@
 SOURCE="$1"
 DRIVE_ROOT="$2"
 
+PROPRIETARY_DOS_INSTALLED_EXIT_CODE=100
```

Comment:
That just moves the detection from either based on a file to based on a directory name. You will need to recognize the possible different directory names as the detection then.

Btw. a downloaded dos will always end up below /var as /usr is not meant for such files. Maybe one day there will be ms-dos packages ;)